.github/workflows/build.yml

```diff
@@ -32,14 +34,25 @@ jobs:
                         php: "8.1"
                         symfony: "^6.4"
                         sylius: "~1.14.0"
+                        database: "mysql"
                         mysql: "8.4"
                         node: "20.x"
                         wkhtmltopdf: "0.12.6-1"
                         state_machine_adapter: "symfony_workflow"
+                    -
+                        php: "8.1"
+                        symfony: "^6.4"
+                        sylius: "~1.14.0"
+                        database: "postgres"
+                        postgres: "15.8"
+                        node: "20.x"
+                        wkhtmltopdf: "0.12.6-1"
+                        state_machine_adapter: "symfony_workflow"
```

Comment:
These additional ones may not be added, they will be enough for one database I guess